docs/server/source/production-deployment-template/node-config-map-and-secrets.rst

```diff
@@ -220,10 +220,9 @@ portal using your admin account, click **APIs** and click on **Integration**
 for the relevant API.
 Scroll to the bottom of the page and click the small link
 in the lower right corner, labelled **Download the NGINX Config files**.
-You'll get a ``.zip`` file.
-Unzip it, then open the ``.conf`` file and the ``.lua`` file.
+Unzip it(if it is a ``zip`` file). Open the ``.conf`` and the ``.lua`` file.
 You should be able to find all the values in those files.
-You have to be careful because it will have values for *all* your APIs,
+You have to be careful because it will have values for **all** your APIs,
```

Comment:
I really did mean to have single asterisks here (italics rather than bold), but it's not that big a deal.